tests/data/cotv4/decision_github_private.json

```diff
@@ -18,6 +18,7 @@
   ],
   "payload": {
     "env": {
+      "MPD001_BASE_REV": "330ea928b42ff2403fc99cd3e596d13294fe8775",
```

Comment:
@escapewindow can't access the repo, I think I should've put the real value here.

Reply:
No worries. Let's drop this change.